src/sections/MainVictories/index.tsx

```diff
@@ -9,7 +9,7 @@ const MainVictories: React.FC = () => {
       coverImage: file(relativePath: { eq: "main-victories-image.png" }) {
         childImageSharp {
           fluid(fit: COVER) {
-            ...GatsbyImageSharpFluid
+            ...GatsbyImageSharpFluid_withWebp
```

Comment:
Just curious, what's the advantage of using `GatsbyImageSharpFluid_withWebp`?

Reply:
It will allow our site to request to the Gatsby GraphQL api a version of the image preferring [Webp format](https://developers.google.com/speed/webp). In that sense, we will take advantage of this format whenever is possible. More condensed information at https://using-gatsby-image.gatsbyjs.org/prefer-webp/ keep in mind that our `gatsby-background-image` uses gatsby-image under the hood.